### ISSUE TITLE
Wiki

### ISSUE THREAD

**Issue by PlayerWithoutName** [storm-enroute/macrogl#44]
Even with examples, some things may be hard to understand. As i'm porting my engine to MacroGL, i'll try to put some pages up there

**Comment by axel22** [comment]
Agreed - we should have more example programs. Note that the infrastructure for documentation already exists on the MacroGL website:

https://github.com/storm-enroute/storm-enroute.github.io/tree/master/macrogl

Specifically, this folder is reserved for examples, and currently has a link to only a single example program:

https://github.com/storm-enroute/storm-enroute.github.io/tree/master/macrogl/docs/0.4

The actual page is hosted here:

http://storm-enroute.com/macrogl/

My plan was to add more examples and explanations there, but I never got around to it. If you would like to add something in the documentation section, that would be much appreciated.

**Comment by axel22** [comment]
Also, I've implemented some infrastructure to automatically upload example programs built with ScalaJS to Github pages, so that there is a sanity check that changes did not break anything:

http://storm-enroute.com/builds/macrogl/2015-10-04_23-59-22_zf8lte/backend-examples-webgl/

At the moment, there is only a single example program set up.
